src/main/controllers/features/limited/addressCorrespondanceSelectorController.ts

```diff
@@ -79,7 +80,8 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
                 acspData.correspondenceAddress = acspData.businessAddress;
                 try {
                     //  save data to mongodb
-                    await postAcspRegistration(session, session.getExtraData(SUBMISSION_ID)!, acspData);
+                    const acspDataService = new AcspDataService();
```

Comment:
As we are going to use this object in both if and else, we can bring the declaration out and use it in both places.